### ISSUE TITLE
413 error caused when Deleting subscriptions v1

### ISSUE THREAD

**Issue by Bento007** [HumanCellAtlas/data-store#2204]
## Software version (git hash, date, branch)
74b216e20ba34156dcd2d1c17469296200dadb6e

## Expected behavior
subscriptions should be deleted

## Actual behavior
500 error returned

## Steps to reproduce the behavior
create a subscription that matches a lot of es indexes then try to delete your query.

### Logs
The request
```
[INFO]	2019-06-12T18:41:37.703Z	f11c8e44-0485-48a2-8e58-b52d8e516e7d	[request] "DELETE /v1/subscriptions/3cae86cf-b279-426b-96a7-6c3a3c9c8874" 169.233.166.18 - "DSSClient" {'replica': 'aws', 'subscription_type': 'elasticsearch'}
```
The generated elasticsearch request
```
[WARNING]	2019-06-12T18:41:37.859Z	f11c8e44-0485-48a2-8e58-b52d8e516e7d	POST https://search-dss-index-staging-bobpbiduntwlsh2yllwchsiypy.us-east-1.es.amazonaws.com:443/dss-staging-aws-docs-b2b1c6291a4aed0e064617118563aff0d00f2d9a,dss-staging-aws-docs-f860fa3a25db54f6378ae2d48733bcf27df72f26,dss-staging-aws-docs-c885a33ce0ce411b14c63a8c1e4918f3b2e619bf,dss-staging-aws-docs-f3c9a0323ecfe21baf5435d0ce7683dec3306f42,dss-staging-aws-docs-1447342716c3c53918d3bb7caa8381071d89846b,dss-staging-aws-docs-bf393fb633e88d97b1ae9bc7467d6efe7a0292f9,dss-staging-aws-docs-de4c5ee2297fcecbd809c8b67fd535c3db8c1103,dss-staging-aws-docs-b731d1269f16889694ae6b6460d7cc70a3aff2dc,dss-staging-aws-docs-bee31647625ae95c0e22d4f58ae9b59ff8421f10,dss-staging-aws-docs-21bc973e378d390892fa67a68c8df916f724a340,dss-staging-aws-docs-d0ec47cb6228138c05ff4c1fb15e303ecd30ca77,dss-staging-aws-docs-787b78592b76b89afe9f7e487c587dcf83b19ab6,dss-staging-aws-docs-4e881ffbaf039c87d2eaca2144affc2a3cfa72bb,dss-staging-aws-docs-bf64daa19e2230eee3585d62331ad8738368a9ab,dss-staging-aws-docs-1887f13fe12c776e5ce3e6cdf4aefacfc6afe914,dss-staging-aws-docs-fe12c6d8d3fcf84694d16f15751f379006b8f36c,dss-staging-aws-docs-ac122fcbe3c854ae0b3b3da01f2931094b9b2980,dss-staging-aws-docs-b9f6760b45c54e4874cfe1371ef069f9998e6e55,dss-staging-aws-docs-c5054c3d0d3062dd7e87d80207499772b3d8f0db,dss-staging-aws-docs-5860c814ce99f624559608874647c8f4e0ddadaf,dss-staging-aws-docs-6e224416a3df2feaa65d756e77b268689394b75e,dss-staging-aws-docs-6eef6648406c333a4035cd5e60d0bf2ecf2606d7,dss-staging-aws-docs-8105f59f6000b7db586b69d8003c79600949d899,dss-staging-aws-docs-181e9229ecf67a397343f45765d579caa56d9b05,dss-staging-aws-docs-6be44a3c0ba535a1e731a2d0fb42811915b72daf,dss-staging-aws-docs-d401626915c41a21f01205aefd6059b1b61c500e,dss-staging-aws-docs-7a8782f84ad2521f658460ab9b311bfe9dc32d6b,dss-staging-aws-docs-afed5204b5fa187a4dde3c85e5864071d3daa906,dss-staging-aws-docs-d31c6063807def100926ddd9480acbaa1cdf2d65,dss-staging-aws-docs-81afb11de85d603a6894e2c940ac1799900ec6ba,dss-staging-aws-docs-c12d54a875aee82b057a00d4fa395a59911e0136,dss-staging-aws-docs-6950e78b112a1553af1c4d7620707e8a9d242649,dss-staging-aws-docs-c7bba59565b14d98e5a87a79b954810dc1901206,dss-staging-aws-docs-c8d0ec8d65101c47c78e657c629626d00dfc3382,dss-staging-aws-docs-496d949369d199e80eeb90684402f2bfac44fa39,dss-staging-aws-docs-7d5f8175fee54b9a4729a31ea5c8a36ec3e6f416,dss-staging-aws-docs-725b97fdd347f295fa4779f2ec4dd33a87dc96e5,dss-staging-aws-docs-d7adc70897d8d6f47834b9275e42474f17f23a94,dss-staging-aws-docs-4425ab190d71d3cdc35361d138cb77008ff8a99e,dss-staging-aws-docs-f3a9527429e4e4baa90c8e64e213353c399f7bba,dss-staging-aws-docs-494311c668563efadd5ce14965a9920c9e73498c,dss-staging-aws-docs-803b4d9d332a5cc64992e944a5d03c1ef9160b28,dss-staging-aws-docs-f1337f22e5d6cf4326fb607a9767f101231b473f,dss-staging-aws-docs-74db9647677ebf4d781760d063d611db8d00d1dc,dss-staging-aws-docs-21db653e55645b79c99f2156b6f4d2b51e05ef6d,dss-staging-aws-docs-c933e7cc16be73f86cfb054c8232cc11dfc76023,dss-staging-aws-docs-3b3fe4bc03f0b8a27aa21290320ea99c1849db4e,dss-staging-aws-docs-4aaecd09e7da8f6c9d20dcfb6041c99d2dc6db17,dss-staging-aws-docs-95ec5e77e5d8bedb7eda2dc6ff02fa4fac06c71f,dss-staging-aws-docs-901a78a62aad374913926dc44285a39e4df6bb0b,dss-staging-aws-docs-934731abb9adea3139428edc401c166fedf7977b,dss-staging-aws-docs-34c936f2790347745bfe311fbeee6a2d906157f6,dss-staging-aws-docs-8adff43212217d90eadb2ba142832ef89f8f2065,dss-staging-aws-docs-f674ee64724be9aca6f34adcdaf4882902935d43,dss-staging-aws-docs-ca7a7246f0e211c48e623d66afa9f11e0353c8b4,dss-staging-aws-docs-f7286833293829f2ea0c1cdb29174d55b71e4171,dss-staging-aws-docs-9ad1bf7b95cc2aefe5f806b29c4f65a97650e243,dss-staging-aws-docs-19617ae3ab23d92b01a9b0bcb3258f9471ad9d6d,dss-staging-aws-docs-8b87a021f47d30b85e4fd9b18077efa3b8889f27,dss-staging-aws-docs-16f3727abe55ea9c2be75d08703f4bf45de9e856,dss-staging-aws-docs-46d135ee372916c98b7745eff4f0dfb87fa794e1,dss-staging-aws-docs-9301e5f013d2123fe885ff0a41517e3dc722ac6b,dss-staging-aws-docs-f5cf41e325e21fd9b45600eefba42e2f87d84680,dss-staging-aws-docs-b65b941f41fc1cc76759519f97148c9d4142210f,dss-staging-aws-docs-c5e31d5915661de4393e3f1489b00ebc4497dd48,dss-staging-aws-docs-af841980463557a7ee16998e4af56a35810142ee,dss-staging-aws-docs-018cb1e54839d9a2ca1ef0986767806259b3e915,dss-staging-aws-docs-8548a32b513217a3c6979762b4a746bb10d450ba,dss-staging-aws-docs-7ae22d0c10a2872087bcba77b0599396d83efb33,dss-staging-aws-docs-17bcbe5c2cb2e935aeccadcf6d690354425c4e14,dss-staging-aws-docs-8dbc20bde223f1d9eb6b4af1fb04a12937b2a206,dss-staging-aws-docs-10f2e59c52361066b13cc12e440e0daaa0374fac,dss-staging-aws-docs-a77c20159ca1f974514b26e679bd14fb371fbf1a,dss-staging-aws-docs-2b74b102522a243ba2b7823359b563504bef65f7,dss-staging-aws-docs-d762c9c868b70b33c6b1700391cff80cb4741215/query/_delete_by_query?conflicts=proceed&refresh=true [status:413 request:0.006s]
```
The resulting error.
```
[ERROR]	2019-06-12T18:41:37.861Z	f11c8e44-0485-48a2-8e58-b52d8e516e7d	Traceback (most recent call last):
File "/var/task/chalicelib/dss/error.py", line 59, in wrapper
return func(*args, **kwargs)
File "/var/task/chalicelib/dss/api/subscriptions.py", line 25, in delete
return subscriptions_v1.delete(uuid, replica)
File "/var/task/chalicelib/dss/util/security.py", line 113, in wrapper
return func(*args, **kwargs)
File "/var/task/chalicelib/dss/api/subscriptions_v1.py", line 169, in delete
_unregister_percolate(es_client, doc_indexes, uuid)
File "/var/task/chalicelib/dss/api/subscriptions_v1.py", line 186, in _unregister_percolate
refresh=True)
File "/opt/python/lib/python3.6/site-packages/elasticsearch/client/utils.py", line 73, in _wrapped
return func(*args, params=params, **kwargs)
File "/opt/python/lib/python3.6/site-packages/elasticsearch/client/__init__.py", line 898, in delete_by_query
doc_type, '_delete_by_query'), params=params, body=body)
File "/opt/python/lib/python3.6/site-packages/elasticsearch/transport.py", line 312, in perform_request
status, headers, data = connection.perform_request(method, url, params, body, ignore=ignore, timeout=timeout)
File "/opt/python/lib/python3.6/site-packages/elasticsearch/connection/http_requests.py", line 90, in perform_request
self._raise_error(response.status_code, raw_data)
File "/opt/python/lib/python3.6/site-packages/elasticsearch/connection/base.py", line 125, in _raise_error
raise HTTP_EXCEPTIONS.get(status_code, TransportError)(status_code, error_message, additional_info)
elasticsearch.exceptions.TransportError: TransportError(413, '')
```
Dispatched
```
[INFO]	2019-06-12T18:41:37.862Z	f11c8e44-0485-48a2-8e58-b52d8e516e7d	[dispatch] "DELETE /v1/subscriptions/3cae86cf-b279-426b-96a7-6c3a3c9c8874" 500 {'replica': 'aws', 'subscription_type': 'elasticsearch'}
```

### Solution
Modify `_unregister_percolate` to limit the number of indexes sent in a single request and send multiple request if needed. 
https://github.com/HumanCellAtlas/data-store/blob/92f200300212db3d061e3d31141ef93e157422d6/dss/api/subscriptions_v1.py#L176-L183
This will prevent a url of excessive size from being sent to ES and causing the 413 error. May also need to apply this to `_register_percolate`

**Comment by DailyDreaming** [comment]
In the meantime, to fix the current staging deploy, I've tried to consolidate the indices into one by deleting them in kibana with:

`DELETE dss-dev-aws-docs-*`
`DELETE dss-dev-gcp-docs-*`

and attempted reindexing with:

`./scripts/admin-cli.py index --replica aws repair`
`./scripts/admin-cli.py index --replica gcp repair`.

It recreated the same ~80 indices though.

**Comment by DailyDreaming** [comment]
Addressed by #2196.
